Automattic/wp-calypso
# Assembler: Show a single category list on the main screen

Related to https://github.com/Automattic/wp-calypso/issues/83279

## Proposed Changes

* Display the category list on the main screen
* Remove the sections screen
* Update the copy of the description of the header & footer

**Screenshots**

| Before | After |
| - | - |
| ![image](https://github.com/Automattic/wp-calypso/assets/13596067/98ae2ecd-036f-46e8-a87a-f2b63b75ea5b) | ![image](https://github.com/Automattic/wp-calypso/assets/13596067/b52ecb93-1208-4432-b6a8-b18f66acb2d6) |
| ![image](https://github.com/Automattic/wp-calypso/assets/13596067/d01f38bf-410a-445e-bcbe-f61a56b1daa8) | ![image](https://github.com/Automattic/wp-calypso/assets/13596067/b526ec4d-dde0-4f25-b46f-419ced0064c7) |
| ![image](https://github.com/Automattic/wp-calypso/assets/13596067/a05ee8e0-8681-487c-8914-6e4e6957dc04) | ![image](https://github.com/Automattic/wp-calypso/assets/13596067/e6924c1c-7ae2-4b80-aa51-2053a00c9b47) |

**Demo**

https://github.com/Automattic/wp-calypso/assets/13596067/54d0ec19-f675-422f-a059-a28197f68ca7

## Testing Instructions



* Go to /setup?siteSlug=your_site
* Continue to Design Picker
* Select “Design your own” to go to the Assembler screen
* On the Assembler screen
  * Ensure you can see the category list on the main screen
  * Ensure you can open & close the category pattern list when you select/deselect any category
  * Ensure the pattern count would be updated when you select any pattern on any category
  * Ensure the copy of the description of the header & footer is updated

## Pre-merge Checklist
